### ISSUE TITLE
Cannot create multiple CTA buttons

### ISSUE THREAD

**Issue by NhatHo** [linkedin/hopscotch#259]
I'm trying to create a callout step with several CTA buttons, but from the API it seems that there's only 1 CTA button can be created per step. If I'm wrong, please let me know how to do that.

If currently there is only 1 CTA button/step, do you think allowing multiple CTA buttons would be worthwhile?

My suggestion would be passing in an array of objects with each object is a CTA button:
```javascript
ctaButtons: [
    {
        label: 'button 1',
        style: 'custom styles', (optional)
        onClick: function() {}  
    }, { 
         ....
    }
]
```

**Comment by zimmi88** [comment]
Yep, as you noted, the current API only supports one CTA button. If you'd need multiple CTAs, you're probably looking at creating your own template for rendering tour bubbles.

I'd anticipate the API around CTA buttons changing in the near future, though I don't know if this will be part of ES6 Refresh or a later milestone. For now, I'm going to leave this as a tracking ticket for supporting multiple CTAs (or more generally figuring things out with respect to buttons).

I hope this helps!
